zolex/doctrine-encrypt-bundle/4.0/config/packages/zolex_doctrine_encryption.yaml

```diff
@@ -0,0 +1,8 @@
+ambta_doctrine_encrypt:
+    # Secret Key path
+    # secret_directory_path: '%kernel.project_dir%'   # Default value
```

Comment:
Please remove "default config" that are in comments. 
You should give a link to the docs instead.